userland/libtock/led.c

```diff
@@ -1,18 +1,20 @@
-#include "led.h"
-#include "tock.h"
+#include "internal/led.h"
+
+// For LEDs, these simply pass through to the syscall wrappers (the functions
+// in the internal folder).
```

Comment:
There's something I'm not understanding. Why did you move the syscall wrappers to the `internal` folder? It's certainly better commented and has the syscall number set up correctly, but why didn't those just go into `led.[c|h]`?